test/verify/check-accounts

```diff
@@ -135,16 +135,36 @@ class TestAccounts(MachineCase):
         m.execute("userdel berta")
         b.wait_not_in_text('#accounts-list', "Berta Bestimmt")
 
-        # Create a locked user
+        # Create a locked user with weak password
         m.execute("sed -i 's/^SHELL=.*$/SHELL=/' /etc/default/useradd")
         b.click('#accounts-create')
         b.wait_visible('#accounts-create-dialog')
         b.set_input_text('#accounts-create-user-name', "jussi")
         b.set_input_text('#accounts-create-real-name', "Jussi Junior")
-        b.set_input_text('#accounts-create-pw1', good_password)
-        b.set_input_text('#accounts-create-pw2', good_password)
+        b.set_input_text('#accounts-create-pw1', "foo")
+        b.set_input_text('#accounts-create-pw2', "foo")
         b.set_checked('#accounts-create-locked', True)
         b.click('#accounts-create-dialog button.apply')
+
+        # Password is weak, lets change it to another weak - this should still not accept
+        b.wait_in_text("#accounts-create-dialog .dialog-error.help-block", "Password quality check failed:")
+        if m.image != "rhel-8-3-distropkg": # Introduced in #14557
+            b.wait_in_text("#accounts-create-dialog .dialog-error.help-block", "Press Create again to use the password anyway.")
```

Comment:
Reading the RHEL documentation, I realized it uses the word `Click`, rather than `Press`
[See 1.3, item 4](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_systems_using_the_rhel_8_web_console/getting-started-with-the-rhel-8-web-console_system-management-using-the-rhel-8-web-console#logging-in-to-the-web-console_getting-started-with-the-rhel-8-web-console)

Reply:
Updated